xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-fontawesome/src/main/resources/IconThemes/FontAwesome.xml

```diff
@@ -288,7 +288,11 @@ disk = floppy-o
 drink = glass
 drive = hdd-o
 email = envelope-o
-emoticon_smile=smile-o
+emoticon_smile= smile-o
+emoticon_grin = smile-o
+emoticon_unhappy = frown-o
+emoticon_tongue = smile-o
+emoticon_wink = smile-o
```

Comment:
This looks wrong. The same icon for several emoticons?

Reply:
I made the choice to map these even though there is no correct alternative in FA.
Considering those icons are really far away from the initial commit, I'll revert those changes. I'd prefer having all icon transformations mapped to FA to not have silk in there anymore, but I guess it can't be done.
See https://jira.xwiki.org/browse/XRENDERING-734 , we discussed a bit about this IRL with manuel and using UTF-8 for emoticons (like those used in CKEditor) might be a solution.
Reverted the approximative icon mappings in https://github.com/xwiki/xwiki-platform/pull/2590/commits/2caaadf479c90111fb501c6714e4fb0b02663a6b :+1: